qemu/tests/tpm_verify_device.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import re
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify the TPM device info inside guest and host.
+    Steps:
+        1. Boot guest with a emulator TPM device or pass through device.
+        2. Check then verify the TPM info inside guest.
+        3. Check then verify the TPM of the OVMF log on the host.
```

Comment:
Suggest change to: 
3. Check and verify TPM/vTPM device info in the OVMF log on host.

Reply:
Thanks, have updated.